common/changes/@snowplow/node-tracker/feature-1098-verify-eid-returned-node-track_2022-08-31-10-09.json

```
@@ -0,0 +1,10 @@
+{
+  "changes": [
+    {
+      "packageName": "@snowplow/node-tracker",
+      "comment": "Add tests verifying that the track method returns event id (eid) in the payload",
```

We usually add the issue number in the changelog to keep the reference.

```suggestion
      "comment": "Add tests verifying that the track method returns event id (eid) in the payload (#1098)",
```